somaticseq.wdl

```diff
@@ -0,0 +1,289 @@
+version 1.0
+
+import "common.wdl" as common
+
+task SomaticSeqParallelPaired {
+    input {
+        String? preCommand
+        String? installDir
+
+        File? classifierSNV
+        File? classifierIndel
+        String outputDir
+        Reference reference
+        File? inclusionRegion
+        File? exclusionRegion
+        IndexedBamFile tumorBam
+        IndexedBamFile normalBam
+        File? mutect2VCF
+        File? varscanSNV
+        File? varscanIndel
+        File? jsmVCF
+        File? somaticsniperVCF
+        File? vardictVCF
+        File? museVCF
+        File? lofreqSNV
+        File? lofreqIndel
+        File? scalpelVCF
+        File? strelkaSNV
+        File? strelkaIndel
+
+        Int threads = 1
+    }
+
+    String toolCommand = if defined(installDir)
+        then installDir + "/somaticseq_parallel.py"
+        else "/opt/somaticseq/somaticseq_parallel.py"
+        # else here points to the location in the docker image
+
+    command {
+        set -e -o pipefail
```

Comment:
This becomes redundant once precommand is removed, as only one bashcommand is run.